### PR TITLE
Razor template added and code removed.

### PR DESCRIPTION
**Pull request by Thyme-Cat**
Removed a lot of old code for the terminal application, such that it wouldn't create a conflict with the Razor template. Should some parts be missed or become relevant, then we can look at the chirp_cli branch and copy the needed bits. 
The chirp-razor template has not experienced modifications. 

Co-authored-by: Maria arnv@itu.dk 
Co-authored-by: Laura llsa@itu.dk

### CODE REVIEW

File: appsettings.Development.json
Comment (HenrijetteOest) [inline appsettings.Development.json:1]: we think this is autogenerated from Razor

**Review by HenrijetteOest — APPROVED** [review]

**Review by tomat-suppe — COMMENTED** [review]
lgtm

**Review by tomat-suppe — APPROVED** [review]
lgtm

**Review by tomat-suppe — APPROVED** [review]
lgtm

**Pull request merged**